orchestra/utils/common_helpers.py

```diff
@@ -33,11 +34,17 @@ def get_relevance_and_completion_changes(old_todo, new_todo):
     # When activity_log is updated, `todo_change = ''`
     # to avoid triggering any slack messages
     todo_change = ''
-    if old_todo.completed != new_todo.completed:
-        todo_change = 'complete' if new_todo.completed else 'incomplete'
-    elif old_todo.skipped_datetime != new_todo.skipped_datetime:
-        todo_change = 'not relevant' \
-            if new_todo.skipped_datetime else 'relevant'
+    if new_todo.status != old_todo.status:
+        if new_todo.status == Todo.Status.COMPLETED.value and \
+                old_todo.status == Todo.Status.PENDING.value:
```

Comment:
the old logic doesn't look at the old value here, in case it wasn't pending before. i think we should preserve that.

Reply:
The old logic did not need to look for old value because we had already checked if new and old value are not same (`old_todo.completed != new_todo.completed`).

Given that completed (True/False) and skipped_datetime (Null/Not Null) field could only have two states, old value could be implicitly inferred from the new value. This is not the case with status field where we can have three types of values (pending/declined/completed).